### DISCUSSION TITLE
Data Preparation for link prediction?

### DISCUSSION THREAD

**Discussion by chocolocked** [stellargraph/stellargraph#1937] [Q&A]
Hi, can somebody point to the place where I can have some guidance on how to process my own custom data to the format that link prediction will be able to use? Thanks so much!

**Comment by CalmLogarithm** [comment]
Some more information would be useful in answering your question:

1. Which algorithm are you using for link prediction?
2. What kind of data are you working with?
3. Do you have previous experience preprocessing data for ML applications?
